CHANGES.md

```diff
@@ -96,6 +96,7 @@
 * S3 implementation of the Beam filesystem (Go) ([#23991](https://github.com/apache/beam/issues/23991)).
 * Support for SingleStoreDB source and sink added (Java) ([#22617](https://github.com/apache/beam/issues/22617)).
 * Added support for DefaultAzureCredential authentication in Azure Filesystem (Python) ([#24210](https://github.com/apache/beam/issues/24210)).
+* Support for read from Cosmos DB Core SQL API [#23610](https://github.com/apache/beam/pull/23610)
```

Comment:
This will go to Beam 2.45.0 now. Maybe need a rebase to fetch the latest README.md

Reply:
Please, validate my changes

Reply:
@Abacn, Do these files also need a license? README? Test? properties?

```
00:33:46 Files with unapproved licenses:
00:33:46  - /home/jenkins/jenkins-slave/workspace/beam_PreCommit_RAT_Commit/src/sdks/java/io/azure-cosmosdb/README.md
00:33:46  - /home/jenkins/jenkins-slave/workspace/beam_PreCommit_RAT_Commit/src/sdks/java/io/azure-cosmosdb/src/test/resources/log4j.properties
00:33:46  - /home/jenkins/jenkins-slave/workspace/beam_PreCommit_RAT_Commit/src/sdks/java/io/azure-cosmosdb/src/test/scala/org/apache/beam/sdk/io/azure/cosmos/CosmosIOTest.scala
```

Reply:
For README and Test, yes; for properties, not need but this is not exempt by test. Maybe just add a license at the head and no harm.

Reply:
Ok, I fixed everything mentioned.